apps/base/traefik-hub/release.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   chart:
     spec:
       chart: traefik-hub
-      version: v2.7.0
+      version: v2.9.0
```

Comment:
Todo: update to 3.0.0 once https://github.com/traefik/traefik-hub-helm-chart/pull/66 is released

Reply:
```suggestion
      version: v3.0.0
```

Reply:
@immanuelfodor A release of the chart may happen this week.

Reply:
Yaaay, thanks you for the info! :pray: